pkgs/development/tools/documentation/adrdox/cross-platform.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index f54da14..912896e 100644
+--- a/Makefile
++++ b/Makefile
+@@ -2,7 +2,7 @@ LIBDPARSE=Dscanner/libdparse/src/dparse/ast.d Dscanner/libdparse/src/dparse/form
+ 
+ all:
+ 	#dmd diff.d terminal.d $(LIBDPARSE)
+-	dmd -m64 doc2.d latex.d cgi.d comment.d stemmer.d dom.d script.d jsvar.d html.d color.d -J. $(LIBDPARSE) -g # -version=std_parser_verbose 
++	dmd doc2.d latex.d cgi.d comment.d stemmer.d dom.d script.d jsvar.d html.d color.d -J. $(LIBDPARSE) -g # -version=std_parser_verbose 
+ 	#dmd -of/var/www/dpldocs.info/locate locate.d  dom.d stemmer.d  cgi -J. -version=fastcgi -m64 -debug
+ locate:
+ 	dmd -of/var/www/dpldocs.info/locate locate.d  dom.d stemmer.d  cgi -J. -version=fastcgi -m64 -debug postgres.d database.d -L-L/usr/local/pgsql/lib
```

Comment:
Is this patch written by you or did you find it upstream somewhere? Is there an issue upstream to fix this? A patch like this ideally doesn't belong in nixpkgs but should be in upstream.